src/PrestaShopBundle/Resources/config/services/core/routing.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.core.routing.controller_stack:
```

Comment:
I'm a bit sad I had to create a new YAML file to add this service but I could not find a relevant file already existing 😞